url_db.py

```diff
@@ -20,13 +20,13 @@ def check_admin_code_is_available (admin_url: str):
 
     return result is None
 
-def insert_url (admin_url: str, ori: str):
+def insert_url (admin_url: str, ori: str, username: str):
     if not check_admin_code_is_available (admin_url):
         raise ValueError ("The chosen short code is already taken. Please choose a different one.")
 
     conn = sqlite3.connect (url_db_path)
     cursor = conn.cursor()
-    cursor.execute ('''INSERT INTO url_mapping (code, url) VALUES (?, ?)''', (admin_url, ori))
+    cursor.execute ('''INSERT INTO url_mapping (code, url, username) VALUES (?, ?, ?)''', (admin_url, ori, username))
```

Comment:
* `'''` is for multiple line comment or string, you can only use `'` here
* Directly inserting values ​​from user without any validation can lead to sql injection or insert unexpected data